Installation instructions on running airflow locally

RoelWKramer opened the pull request:
To get volumes mounted in pods you need some sort of local storage. This collides with the whole concept of kubernetes. Now a pod is bound to a specific host. For specific purposes, like ours, this is ok. But it also means it was not straightforward to get it up and running with PersistentVolumes, VolumeClaims and StorageClasses and then combine it all together in a helm chart.

Now the volumes have to be configured "manually", before the airflow helm chart is deployed.

RoelWKramer committed:
Installation instructions on running airflow locally

peteradrichem reviewed (approved):
Werkt onder Kubernetes

RoelWKramer committed:
Create a volume bound to the dags directory on the dev machine

RoelWKramer committed:
Create volume manually

RoelWKramer committed:
Mount volume in pod

Volume is mounted in pods, to pass this directory dynamically,
strings are substituted in volume.yaml.tpl.

RoelWKramer commented on airflow-local/templates/volume.yaml.tpl:
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```
This is a bit unfortunate, but there was just no way to get hostPath working by creating a Chart and then get it mounted in the pod. Therefor this is a template which gets is "variables" from install.sh.
